src/types.ts

```diff
@@ -8,8 +8,7 @@ export type DapiName = string;
 export type PrivateKey = string;
 export type DataFeedId = EvmId;
 export type ChainId = string;
-export type DApiName = string;
-export type Provider = string;
+export type ProviderName = string;
```

Comment:
The `Provider` makes me feel it's something from ethers.

Reply:
It feels better as `ProviderName` 👍